authentik/root/settings.py

```diff
@@ -207,7 +207,7 @@
 
 REST_FRAMEWORK = {
     "DEFAULT_PAGINATION_CLASS": "authentik.api.pagination.Pagination",
-    "PAGE_SIZE": 100,
+    "PAGE_SIZE": CONFIG.get("pagination.page_size", 100),
```

Comment:
Since the configuration was moved out to the DB, maybe want to revert this?
```suggestion
    "PAGE_SIZE":  100,
```